proto/osmosis/superfluid/query.proto

```diff
@@ -70,6 +70,15 @@ service Query {
                                    "superfluid_delegations/{delegator_address}";
   }
 
+  // Returns all the superfluid poistions for a specific delegator
+  rpc SuperfluidOSMODelegationsByDelegator(
```

Comment:
This is a mouthful haha. Any chance we can think of more abbreviated nomenclature.

Reply:
he we can think of better names. How do u think about the logic

Reply:
@mattverse want to create  a separate query (as i understant). Refer to pr https://github.com/osmosis-labs/osmosis/pull/1539

Reply:
Should change to _SuperfluidEquivilentDelegationsByDelegator_?